src/System.Diagnostics.Process/tests/ProcessTest_ConsoleApp/ProcessTest_ConsoleApp.cs

```diff
@@ -7,6 +8,7 @@ class Program
     {
         static int Main(string[] args)
         {
+            Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
             try
```

Comment:
While I'm fine with this change I do think we should also fix the Process tests to not expect the exact string. Would you mind updating the tests to look for particular keywords. For example in Process_SyncErrorStream I would change the assert to check for only parts like "Intentional Exception thrown" not requiring the exact exception message. We should even consider making the messages a little more unique and less error prone to things like loc and missing symbols, something like "[ProcessTestConsoleApp_IntentionalError]".